app/models/requests/requestable.rb

```diff
@@ -40,7 +40,7 @@ def initialize(bib:, holding: nil, item: nil, location: nil, patron:)
     end
 
     delegate :pick_up_location_id, :pick_up_location_code, :item_type, :enum_value, :cron_value, :item_data?,
-             :temp_loc?, :in_resource_sharing?, :on_reserve?, :inaccessible?, :hold_request?, :enumerated?, :item_type_non_circulate?, :partner_holding?,
+             :temp_loc_other_than_resource_sharing?, :on_reserve?, :inaccessible?, :hold_request?, :enumerated?, :item_type_non_circulate?, :partner_holding?,
```

Comment:
I'm a little bit confused with this replacement.
`temp_loc? && !in_resource_sharing?`

Reply:
For me it took a long time to understand what `temp_loc? && !in_resource_sharing?` was representing, and I found it confusing that neither is used by itself, but that same chunk was in several places. Also reek was complaining about how much `temp_loc?` was being repeated